packages/cli/src/commands/destroy/component/__tests__/component.test.js

```diff
@@ -1,34 +1,42 @@
 globalThis.__dirname = __dirname
-jest.mock('fs')
-jest.mock('../../../../lib', () => {
+vi.mock('fs-extra', async () => {
+  const memfs = await import('memfs')
   return {
-    ...jest.requireActual('../../../../lib'),
+    default: memfs.fs,
+  }
+})
```

Comment:
Can't you just do `vi.mock('fs-extra', import('memfs')`?

Reply:
Yeah this is totally possible and I'm happy to switch to it. I'll ensure they're all consistent before this is ready.

Reply:
I've centralised the mock into the `__mocks__` and reduced the number of copies of the above code.